addons/website_sale/models/website.py

```diff
@@ -1,21 +1,26 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
+import base64
+import json
+import logging
+import random
 
 from odoo import SUPERUSER_ID, api, fields, models
-from odoo.exceptions import UserError
+from odoo.addons.iap.tools import iap_tools
+from odoo.exceptions import AccessError, UserError
 from odoo.http import request
 from odoo.osv import expression
-from odoo.tools import lazy, ormcache
+from odoo.tools import file_open, lazy, ormcache
 from odoo.tools.translate import LazyTranslate, _
 
-
+logger = logging.getLogger(__name__)
 _lt = LazyTranslate(__name__)
 
 CART_SESSION_CACHE_KEY = 'sale_order_id'
+DEFAULT_OLG_ENDPOINT = 'https://olg.api.odoo.com'
```

Comment:
I guess this should not be redefined... there is a `_OLG_api_rpc` method on website... maybe this already does what you want.
It already feels a bit weird that there are two distinct overrides for web_editor and website... but supposing that distinction is intentional, since this is in the website configurator,  I'd say it should rely on the website one.